DALI.tex

```diff
@@ -1310,6 +1311,13 @@ \subsubsection{Additional Information}
 
 \section{Changes}
 
+\subsection{PR-DALI-1.2}
+\begin{itemize}
+\item Clarified that truncation indicated by OVERFLOW can occur independent of
+MAXREC
+\item added new xtypes: moc, shape, multipolygon
+\end{itemize}
```

Comment:
Should this include that VOSI-availability has been made optional and VOSI-capabilities is only required for registered services?

Reply:
thnx for reminder